_docs/install/prerequisites/devicepresentation.md

```diff
@@ -7,51 +7,14 @@ module: install/prerequisites/devicepresentaion.md
 
 # Device presentation
 
-StorageOS requires one of two runtime requirements to present virtual block devices
-from userspace. While not mandatory, the NBD aproach does offer significantly
-better performance over the loop-device alternative.
+StorageOS is developed using Linux-IO (LIO) Target. An open-source implementation of the SCSI target. StorageOS uses a 
+software interface for Unix-like computer operating systems that lets non-privileged users create their own file systems without editing kernel code.
 
-## Network Block Device
-
-NBD (Network Block Device) is a Linux kernel module, available on many
-distributions, that allows block devices to be run in userspace. It is not a
-requirement for StorageOS to run, but improves performance significantly and
-will be used in preference if available. To enable the module and increase the
-number of allowable devices, run:
-
-```bash
-sudo modprobe nbd nbds_max=1024
-```
-
-To ensure the NBD module is loaded on reboot:
-
-1. Add the following line to `/etc/modules`
-
-    ```text
-    nbd
-    ```
+However, LIO is supported by most of the kernels available nowadays, some distributions have left the kernel module out of the main kernel image. For those, distributions with support,
+will be required to enable kernel modules. 
 
-1. Add the following module configuration lines in `/etc/modprobe.d/nbd.conf`
+Check the [os distribution support page](/docs/reference/os_support) to validate what OS can be used and how to enable LIO.
```

Comment:
"os" => "OS"
"what" => "which"

Reply:
Thank you! Done https://github.com/storageos/storageos.github.io/pull/148/commits/85f8b90c9002e67efe6eff881dff4b73ed98fad3